apps/block_scout_web/priv/gettext/en/LC_MESSAGES/default.po

```diff
@@ -19,7 +19,7 @@ msgstr[0] ""
 msgstr[1] ""
 
 #, elixir-format
-#: lib/block_scout_web/views/address_token_balance_view.ex:9
+#: lib/block_scout_web/views/address_token_balance_view.ex:10
```

Comment:
That's weird, why did translations change in this commit?

Reply:
not sure, but the tests were failing https://github.com/celo-org/blockscout/runs/8250389979?check_suite_focus=true

possibly something to do with caching - there's a new dependency which caused a cache rebuild of all apps, perhaps this change would only have been picked up previously when a code file directly in the `block_scout_web` app changed